### ISSUE TITLE
Create our own "getting started" etc instead of importing CAPI docs

### ISSUE THREAD

**Issue by hackeramitkumar** [kubernetes-sigs/cluster-api-provider-aws#4067]
/kind feature

**Describe the solution you'd like**
I will write CAPA getting-started. It will fix broken URLs. And will be more user friendly to new CAPA users.

**Environment:**

- Cluster-api-provider-aws version: 
- Kubernetes version: (use `kubectl version`): 
- OS (e.g. from `/etc/os-release`): 
-
/kind documentation

**Comment by Ankitasw** [comment]
I think we are embedding docs from CAPI such that we avoid duplication of content.
Is there no other alternative to still use embed github feature such that all the links are fixed?

**Comment by hackeramitkumar** [comment]
@Ankitasw yes we have one way of fixing it. We have to restructure our book directories. We have a discussion on this topic in slack channel you can check it there. I have mentioned all the possible ways of fixing that. 
https://kubernetes.slack.com/archives/CD6U2V71N/p1675510418951049

**Comment by Ankitasw** [comment]
/triage accepted
/priority important-soon


@hackeramitkumar thanks for raising this. You could go ahead and assign it to yourself.
